Sources/Sentry/SentryProfiler.mm

```diff
@@ -193,75 +194,32 @@
  * didn't occur within the profile time.
  */
 NSArray<SentrySerializedMetricReading *> *
-processFrameRenders(SentryFrameInfoTimeSeries *frameInfo, SentryTransaction *transaction)
+sliceGPUData(SentryFrameInfoTimeSeries *frameInfo, SentryTransaction *transaction)
```

Comment:
Here's where the aforementioned consolidation of slicing frame renders and refresh rates is. Since they're not written in a uniform data structure with the same keys, they can all be handled with the same logic instead of two different functions. `processFrameRenders` used to not only slice the data to transaction bounds, but also had to calculate frame durations; that's now done in the frames tracker. After that, it does the same thing as `processFrameRenders` did, so they are combined in a single function now named `slicedGPUData`.